Wireframe/index.html

```diff
@@ -9,24 +9,46 @@
   <body>
     <header>
       <h1>Wireframe</h1>
-      <p>
-        This is the default, provided code and no changes have been made yet.
+      <p id = "welcome">
```

Comment:
Can you check the Wireframe in the exercise instructions? The page title and summary is not aligned in the same way as shown in the wireframe.